spec/lib/alerts/medicine_article_monitor_spec.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require "#{Rails.root}/lib/alerts/medicine_article_monitor"
+
+def mock_mailer
+  OpenStruct.new(deliver_now: true)
+end
+
+describe MedicineArticleMonitor do
+  let(:mntor) { described_class.new }
+  let(:course1) { create(:course, slug: 'slug-one') }
+  let(:course2) { create(:course, slug: 'slug-two') }
+  let(:student1) { create(:user, username: 'alice', email: 'student1@example.edu') }
+  let(:med_article) { create(:article, title: 'my med title 2', mw_page_id: 222) }
+  let(:content_expert) { create(:user, greeter: true) }
+  let(:med_training_module) { create(:training_module, slug: 'editing-medical-topics') }
```

Comment:
I think it's better here to use the same `TrainingModule.load_all` method from the other specs for setting up the training module content for the tests rather than using a factory — especially per my comments above, since we should be specifying the relevant modules via their static IDs. That will also make it so that no changes to other tests are necessary.

Reply:
I have reverted to that.